mmtrack/datasets/pipelines/formatting.py

```diff
@@ -356,23 +356,29 @@ def __call__(self, results):
 class SeqReIDFormatBundle(SeqDefaultFormatBundle):
```

Comment:
Name the class as `ReIDFormatBundle`. The "Seq" means there are pair images for training. 
If you do not use the triplet loss, it is just a normal formatting.